plugins/woocommerce/tests/legacy/unit-tests/checkout/checkout.php

```
@@ -313,4 +313,54 @@ public function test_create_order_for_variation_product() {
+			$child = wc_get_product( $child_id );
+			if ( '' === ( $child->get_variation_attributes()['attribute_pa_number'] ?? null ) ) {
+				$any_number_variation_id = $child_id;
+				$any_number_variation    = $child;
+				break;
+			}
+		}
+		$this->assertGreaterThan( 0, $any_number_variation_id, 'Expected a variation with an "any" pa_number attribute.' );
+
+		// Build cart attributes: use the variation's fixed values for non-number attributes, supply an arbitrary value for
+		// any remaining "any" attributes, and the customer's chosen value (1) for the "any" pa_number attribute under test.
+		$attrs          = $any_number_variation->get_variation_attributes();
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$attrs'. (undefined)

(UnusedLocalVariable)